pgpy/constants.py

```diff
@@ -382,9 +382,12 @@ def tune_count(self):
         # GnuPG tunes for about 100ms, so we'll do that as well
         _TIME = 0.100
         ct = int(len(htd) * (_TIME / (end - start)))
-        c1 = ((ct >> (ct.bit_length() - 5)) - 16)
-        c2 = (ct.bit_length() - 11)
-        c = ((c2 << 4) + c1)
+        if ct.bit_length() < 11:
```

Comment:
You're not "doing the math correctly," this is the encoding step. You're encoding the value incorrectly which will result in incorrect values being decoded by any recipient. If the resulting value is too small to properly encode, that means one of two things:

1. The math that's wrong is on line 384, not any of this
2. The approach of copying GPG's method of tuning the count was flawed

Honestly I think we'd be in much better shape if we just max out the tuned count by default like you suggest. I like that idea much better than this lol

Reply:
I still think that the fix here was the correct version of what this code was trying to do -- it is the correct encoding of a number that was very small into the weird 8-bit-float approximation OpenPGP's S2K count.  But i agree that all this stuff is silly and complex and probably more likely to capture transient CPU contention than any really slow hardware.

And, even if the calibration is done on slow hardware, it's generally supposed to be a defense against an attacker whose hardware we don't know, but is probably at least as powerful as a 10-year-old laptop, which can plausibly max out the counter anyway.

So please consider merging #301 instead of this patch :)